dashboard/app/models/user.rb

```diff
@@ -2345,9 +2345,8 @@ def has_pilot_experiment?(pilot_name)
   private def soft_delete_channels
     return unless user_storage_id
 
-    channel_ids = PEGASUS_DB[:storage_apps].
-      where(storage_id: user_storage_id).
-      map(:id)
+    user_storage_apps = StorageApps.new(user_storage_id)
+    channel_ids = user_storage_apps.get_all_storage_ids
```

Comment:
Should we rename `channel_ids` to reduce confusion here?

Reply:
Yes! Another perfect example of where the use of the term "channel" is confusing. I'll swap this out now, but I have another ticket https://codedotorg.atlassian.net/browse/LP-2272 where I intend to do a more focused sweep of the names we're using